pennylane/_qubit_device.py

```diff
@@ -754,8 +754,9 @@ def statistics(
             elif obs.return_type is Probability:
                 # Appends a result of shape (2**len(obs.wires), num_bins,)
                 # if bin_size is not None else (2**len(obs.wires),)
+                wires = obs.wires or self.wires
```

Comment:
Think we could move this logic inside the `self.probability` method? That might be a way to satisfy pylint.

Reply:
Done!